server/lib/routes/subscription.js

```diff
@@ -3,18 +3,9 @@
 const S = require('fluent-json-schema')
 
 const subscriptionMap = new Map()
```

Comment:
Not sure if we need `subscriptionMap ` anymore since there is only one subscription type which is `expo`. We probably can extract the value directly like `request.body['token']`. The `type` is already being validated.